src/main/java/org/myrobotlab/framework/ServiceType.java

```diff
@@ -32,28 +33,57 @@
 
   private static final long serialVersionUID = 1L;
 
-  String name;
-  String simpleName;
-  String link;
-  String license;// = "Apache";
-  Boolean isCloudService = false;
-  Boolean requiresKeys = false;
-  Boolean includeServiceInOneJar = false;
-
-  public String getLink() {
-    return link;
-  }
-
-  public void setLink(String link) {
-    this.link = link;
-  }
-
-  String state = null;
-  Integer workingLevel = null;
+  public static ServiceType fromMetaData(MetaData meta) {

```

Comment:
Whats the difference between this class and MetaData ..  I can't see any justification for having both.

Reply:
Ha !  356 files diff - and refusal to look at it.  Yes there is no difference, that's what I have been saying in my large pull request.  Merging the two means a large amount of file imports will change.  I UN-MERGED the two classes to satisfy the small pr .. you can not have both. Pick one or the other.